Partners/support-escalation-process-for-partners.md

```diff
@@ -113,10 +117,12 @@ Our Support team is on call 24x7 with Engineers and Leads on shifts (i.e., rotat
 * **Executive Overhead**: CenturyLink Cloud Executive Staff
 
 ### Security Issues
-Security Incidents follow a standard Incident Process. Incident Response team coordinate with legal, marketing, and executive team.
+Security Incidents follow a standard Incident Process. Incident Response teams coordinate with legal, marketing, and executive team.
 * If an incident is security related, security SMEs are engaged in the same manner as other SMEs.
 * If an incident is determined to be a security breach of the CenturyLink Cloud environment, post-incident response and communication will be led by the Security.
 
+* [Learn More About What to Expect form a Security Issue](../what-can-you-expect-from-tier-3-on-a-security-issue.md)
```

Comment:
@BryGittyUp change link to `../Support/what-can-you-expect-from-tier-3-on-a-security-issue.md`